test/powershell/Language/Operators/ComparisonOperator.Tests.ps1

```diff
@@ -82,6 +82,22 @@ Describe "ComparisonOperator" -Tag "CI" {
         param($lhs, $operator, $rhs)
         Invoke-Expression "$lhs $operator $rhs" | Should -BeFalse
     }
+
+    It "Should be <result> for <lhs> <operator> <rhs>" -TestCases @(
```

Comment:
Please use better name. It should reflect we test backtick.

Reply:
Included backtick in title